class-two-factor-core.php

```diff
@@ -526,29 +536,33 @@ public static function show_two_factor_login( $user ) {
 	 * @since 0.1-dev
 	 */
 	public static function backup_2fa() {
-		if ( ! isset( $_GET['wp-auth-id'], $_GET['wp-auth-nonce'], $_GET['provider'] ) ) {
+		$wp_auth_id = filter_input( INPUT_GET, 'wp-auth-id', FILTER_SANITIZE_NUMBER_INT );
+		$nonce      = filter_input( INPUT_GET, 'wp-auth-nonce', FILTER_SANITIZE_STRING );
+		$provider   = filter_input( INPUT_GET, 'provider', FILTER_SANITIZE_STRING );
+
+		if ( ! $wp_auth_id || ! $wp_auth_nonce || ! $provider ) {
```

Comment:
Looks like `$wp_auth_nonce` should be `$nonce` here

Reply:
Thank you so much for finding this. Looks like we didn’t catch it during the initial code review. And I couldn’t figure out why the current code was failing on my test site...